tests/tests_torch/methods/test_torch_mls.py

```diff
@@ -20,30 +20,39 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-import numpy as np
+import pytest
+from torch.utils.data import DataLoader
 
 from oodeel.methods import MLS
-from tests import almost_equal
-from tests import generate_data_tf
-from tests import generate_model
-
-
-def test_isood():
-    input_shape = (32, 32, 3)
+from tests.tests_torch import ComplexNet
+from tests.tests_torch import generate_data
+from tests.tests_torch import generate_data_torch
+
+
+@pytest.mark.parametrize(
+    ("backend", "input_shape"),
+    [("torch", (3, 32, 32))],
+    ids=["[torch] test DKNN"],
+)
+def test_dknn(backend, input_shape):
```

Comment:
There are several references to DKNN here that should be replaced with MLS instead: function's name, docstring, OODModel instance's name